Test/Integration/Model/Transaction/OrderTest.php

```diff
@@ -15,172 +15,128 @@
+        $order = OrderBuilder::anOrder()->build();
+        $result = $this->taxjarOrder->build($order);
 
-        $this->assertEquals(27.0, $result['line_items'][0]['unit_price'], 'Invalid provider');
+        $this->assertEquals(10.0, $result['line_items'][0]['unit_price'], 'Invalid provider');
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple.php
-     */
     public function testOrderNoShipping()
     {
-        $order = $this->order->loadByIncrementId('100000002');
-        $result = $this->transactionOrder->build($order);
+        $order = OrderBuilder::anOrder()
+            ->withProducts(ProductBuilder::aVirtualProduct())
+            ->build();
+
+        $result = $this->taxjarOrder->build($order);
 
         $this->assertEquals(0, $result['shipping'], 'Invalid shipping amount');
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple_AU.php
-     */
-    public function testAddressAU()
+    public function testOrderNotSyncable()
     {
-        $order = $this->order->loadByIncrementId('100000003');
-        $result = $this->transactionOrder->isSyncable($order);
+        $order = OrderBuilder::anOrder()
+            ->withCustomer(
+                CustomerBuilder::aCustomer()
+                    ->withAddresses(
+                        AddressBuilder::anAddress('en_AU')->asDefaultBilling()->asDefaultShipping()
+                    )
+            )
+            ->build();
 
-        $this->assertFalse($result, 'Non-US order sync');
-    }
+        $result = $this->taxjarOrder->isSyncable($order);
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple_AUD.php
-     */
-    public function testCurrencyAUD()
-    {
-        $order = $this->order->loadByIncrementId('100000004');
-        $result = $this->transactionOrder->isSyncable($order);
-
-        $this->assertFalse($result, 'Non-USD order sync');
+        $this->assertFalse($result, 'Non-US order sync');
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple_shipping.php
-     */
     public function testOrderShipping()
     {
-        $order = $this->order->loadByIncrementId('100000005');
-        $result = $this->transactionOrder->build($order);
+        $order = OrderBuilder::anOrder()->build();
+        $order->setShippingAmount(20.0);
+
+        $result = $this->taxjarOrder->build($order);
 
-        $this->assertEquals(5.0, $result['shipping'], 'Invalid shipping amount');
+        $this->assertEquals(20.0, $result['shipping'], 'Invalid shipping amount');
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple_customer.php
-     */
     public function testExemptCustomer()
```

Comment:
The name of this test is also deceptive as we are testing that the customer id on the payload is correct and not testing anything regarding the customers exemption status.